### ISSUE TITLE
[Bug] about learning rate config

### ISSUE THREAD

**Issue by qierrrrrrr** [open-mmlab/mmrotate#778]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmrotate/issues) and [Discussions](https://github.com/open-mmlab/mmrotate/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmrotate.readthedocs.io/en/1.x/notes/4_faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmrotate) or [latest version (1.x)](https://github.com/open-mmlab/mmrotate/tree/dev-1.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmrotate

### Environment

sys.platform: linux
Python: 3.8.16 | packaged by conda-forge | (default, Feb  1 2023, 16:01:55) [GCC 11.3.0]
CUDA available: True
GPU 0: NVIDIA RTX A5000
CUDA_HOME: /home/cuda-11.3
NVCC: Cuda compilation tools, release 11.3, V11.3.109
GCC: gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0
PyTorch: 1.12.0+cu116
PyTorch compiling details: PyTorch built with:
  - GCC 9.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.6.0 (Git Hash 52b5f107dd9cf10910aaa19cb47f3abf9b349815)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.6
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86
  - CuDNN 8.5  (built against CUDA 11.7)
    - Built with CuDNN 8.3.2
  - Magma 2.6.1
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.6, CUDNN_VERSION=8.3.2, CXX_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -DEDGE_PROFILER_USE_KINETO -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.12.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=OFF, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF,

TorchVision: 0.13.0+cu116
OpenCV: 4.7.0
MMCV: 1.6.2
MMCV Compiler: GCC 9.3
MMCV CUDA Compiler: 11.6
MMRotate: 0.3.4+unknown

### Reproduces the problem - code sample

# evaluation
evaluation = dict(interval=1, metric='mAP')
# optimizer
optimizer = dict(type='SGD', lr=0.0025, momentum=0.9, weight_decay=0.0001)
optimizer_config = dict(grad_clip=dict(max_norm=35, norm_type=2))
# learning policy
lr_config = dict(
    policy='step',
    warmup='linear',
    warmup_iters=500,
    warmup_ratio=1.0 / 3,
    step=[8, 11])
runner = dict(type='EpochBasedRunner', max_epochs=12)
checkpoint_config = dict(interval=1)

### Reproduces the problem - command or script

python tools/train.py

### Reproduces the problem - error message

2023-03-21 10:21:16,953 - mmrotate - INFO - Epoch [1][500/4867] lr: 4.993e-03, eta: 2:59:23, time: 0.182, data_time: 0.004, memory: 8263, loss_rpn_cls: 0.1518, loss_rpn_bbox: 0.2920, loss_cls: 0.3903, acc: 89.6680, loss_bbox: 0.2823, loss: 1.1164, grad_norm: 5.2866
2023-03-21 10:21:25,892 - mmrotate - INFO - Epoch [1][550/4867] lr: 5.000e-03, eta: 2:58:36, time: 0.179, data_time: 0.004, memory: 8263, loss_rpn_cls: 0.1499, loss_rpn_bbox: 0.2143, loss_cls: 0.3445, acc: 91.2969, loss_bbox: 0.2542, loss: 0.9630, grad_norm: 4.1588
2023-03-21 10:21:34,847 - mmrotate - INFO - Epoch [1][600/4867] lr: 5.000e-03, eta: 2:57:57, time: 0.179, data_time: 0.004, memory: 8263, loss_rpn_cls: 0.1078, loss_rpn_bbox: 0.2029, loss_cls: 0.3299, acc: 91.0312, loss_bbox: 0.2801, loss: 0.9206, grad_norm: 4.3258

### Additional information

Actually, I want the learning rate to be 2.5e-03, but no matter how I change the code, the final learning rate is 5e-03 in the first stage

**Comment by zytx121** [comment]
Hi @qierrrrrrr, this is caused by inheritance of configuration files, for example:
https://github.com/open-mmlab/mmrotate/blob/7755aa53b6e2d9c61fd765e67a60daa7ed8a1e05/configs/oriented_rcnn/oriented_rcnn_r50_fpn_1x_dota_le90.py#L146
will overwrite
https://github.com/open-mmlab/mmrotate/blob/7755aa53b6e2d9c61fd765e67a60daa7ed8a1e05/configs/_base_/schedules/schedule_1x.py#L4

**Comment by 123456hxh** [comment]
I'm having a problem and I don't know how to fix it, thanks
train.py: error: the following arguments are required: config

**Comment by qierrrrrrr** [comment]
> 

thank you,I have solved this problem following your advice

**Comment by qierrrrrrr** [comment]
> Hi @qierrrrrrr, this is caused by inheritance of configuration files, for example:
> 
> https://github.com/open-mmlab/mmrotate/blob/7755aa53b6e2d9c61fd765e67a60daa7ed8a1e05/configs/oriented_rcnn/oriented_rcnn_r50_fpn_1x_dota_le90.py#L146
> 
> 
> will overwrite
> https://github.com/open-mmlab/mmrotate/blob/7755aa53b6e2d9c61fd765e67a60daa7ed8a1e05/configs/_base_/schedules/schedule_1x.py#L4

thank you,I have solved problem with your advice

**Comment by qierrrrrrr** [comment]
> I'm having a problem and I don't know how to fix it, thanks train.py: error: the following arguments are required: config

maybe you need command like this: python tools/train.py config=configs/XXXX/XXXXX.py

**Comment by 123456hxh** [comment]
I thank you from the bottom of my heart @qierrrrrrr
